src/PageLayout/useStickyPaneHeight.ts

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import {useInView} from 'react-intersection-observer'
+
+/**
+ * Calculates the height of the sticky pane such that it always
+ * fits into the viewport even when the header or footer are visible.
+ */
+// TODO: Handle sticky header
```

Comment:
@vdepizzol How you feel about adding a prop called `stickyTop` (open to other name ideas) that would allow you to push the side pane down to make room for sticky headers if necessary?

Reply:
I think it makes sense. One thing to keep in mind is that this needs to remain functional as we move to `rem` for all of our UI.

Reply:
Sounds good. I'll tackle this in a separate PR. Created an issue for it here: https://github.com/github/primer/issues/1200